DataFormats/DeepFormats/interface/ChargedCandidateFeatures.h

```diff
@@ -0,0 +1,82 @@
+#ifndef DataFormats_DeepFormats_ChargedCandidateFeatures_h
+#define DataFormats_DeepFormats_ChargedCandidateFeatures_h
+
+namespace deep {
```

Comment:
please rename rename "deep" namespace to something with btag btv in the name.

Reply:
changed to btagbtvdeep in c873f8a2faa84ccd40a3ffc62706aac06bb7d7ab